packages/core/spec/fixtures/templates/handlebars-json/helpers.js

```diff
@@ -0,0 +1,309 @@
+  return ret
+}
+
+/**
+ * Return the JSON representation of a Block
+ * @param {AbstractNode} node the node to process
+ */
+function getBlockInfo (node) {
+  const ret = getAbstractBlockInfo(node)
+  ret.source = node.getSource()
+  ret.sourceLines = node.getSourceLines()
+  ret.content2 = node.getContent()
+  return ret
+}
+
+/**
+ * Return a JSON representation of the ListNode.
+ * If it is a dlist, it is decomposed into special object
+ * with the terms and definitions exposed via attributes
+ * @param {AbstractNode} node the node to process
+ */
+function getListInfo (node) {
+  const theItems = []
+  node.getItems().forEach((item) => {
+    if (node.getNodeName() === 'dlist') {
+      const theTerms = []
+      item[0].forEach((term) => {
+        theTerms.push(term.getText())
+      })
+      const theDescription = {
+        ...getCommonNodeInfo(item[1]), ...getAbstractBlockInfo(item[1])
+      }
+      theDescription.content = item[1].getContent()
+      theItems.push({
+        terms: theTerms,
+        isDescriptionBlock: !isString(item[1].getText()),
+        descText: item[1].getText(),
+        description: theDescription.isBlock ? theDescription : theDescription.text
+      })
+    } else {
+      const theItem = getAbstractBlockInfo(item)
+      theItem.text = item.getText()
+      theItem.hasText = item.hasText()
+      theItem.marker = item.getMarker()
+      theItem.content2 = item.getContent()
+      theItems.push(theItem)
+    }
+  })
+  const ret = getAbstractBlockInfo(node)
+  ret.hasItems = node.hasItems()
+  ret.items = theItems
+  return ret
+}
+
+/**
+ * Return a JSON representation of a given node
+ * @param {AbstractNode} node the node to convert
+ */
+function toJSON (node) {
+  let ret = getCommonNodeInfo(node)
+  if (ret.isInline) {
+    ret.document = getDocumentInfo(node.getDocument())
+    ret.text = node.getText()
+    ret.type = node.getType()
+    ret.target = node.getTarget()
+  } else {
+    ret.blockType = getBlockType(node)
+    if (ret.blocType === 'document') {
+      ret = { ...ret, ...getDocumentInfo(node) }
+      ret.content2 = node.getContent()
+    } else {
+      ret.document = getDocumentInfo(node.getDocument())
+      if (ret.blocType === 'section') {
+        ret = { ...ret, ...getSectionInfo(node) }
+      } else if (ret.blockType === 'list') {
+        ret = { ...ret, ...getListInfo(node) }
+      } else {
+        ret = { ...ret, ...getBlockInfo(node) }
+      }
+    }
+  }
+  return ret
+}
+
+/**
+ * Return a string with the HTML attribute for the given value.
+ * If the value is empty, an empty string is returned
+ * @param {string} name the name of the attribute
+ * @param {value} value the value of the attribute
+ */
+function toAttribute (name, value) {
+  return value !== undefined && value.length !== 0 ? ` ${name}="${value}"` : ''
+}
+
+function configure (options) {
+  options = options || {}
+  const handlebars = options.engine || require('handlebars')
```

Comment:
With https://github.com/asciidoctor/asciidoctor.js/pull/926 it will be:

```suggestion
  options = options || { handlebars: { } }
  const handlebars = options.handlebars.environment || require('handlebars')
```

Reply:
@henriette-einstein sorry since #926 is not yet merged, the property `handlebars` can be `undefined` :sweat_smile:

Reply:
I have added an additional check for that.